src/label_nodes/CCLabelBMFont.cs

```diff
@@ -271,25 +274,30 @@ public CCLabelBMFont(string str, string fntFile, float width, CCTextAlignment al
         {
         }
 
-        public CCLabelBMFont(string str, string fntFile, float width, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment, 
+        public CCLabelBMFont(string str, string fntFile, float width, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment,
             CCPoint imageOffset, CCTexture2D texture)
             : this(str, fntFile, new CCSize(width, 0), hAlignment, vAlignment, imageOffset, texture)
         {
         }
 
-        public CCLabelBMFont(string str, string fntFile, CCSize dimensions, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment, 
+        public CCLabelBMFont(string str, string fntFile, CCSize dimensions, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment,
             CCPoint imageOffset, CCTexture2D texture)
         {
             InitCCLabelBMFont(str, fntFile, dimensions, hAlignment, vAlignment, imageOffset, texture);
         }
 
-        protected void InitCCLabelBMFont(string theString, string fntFile, CCSize dimensions, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment, 
+        protected void InitCCLabelBMFont(string theString, string fntFile, CCSize dimensions, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment,
             CCPoint imageOffset, CCTexture2D texture)
         {
             Debug.Assert(FontConfiguration == null, "re-init is no longer supported");
             Debug.Assert((theString == null && fntFile == null) || (theString != null && fntFile != null),
                 "Invalid params for CCLabelBMFont");
 
+            if (!((theString == null && fntFile == null) || (theString != null && fntFile != null)))
+            {
+                Debug.WriteLine("ERROR");
```

Comment:
Can we use CCLog.Log here instead of Debug and a more descriptive error if possible.